### ISSUE TITLE
clarify cross-val assignment from 05?

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/gadsdc#14]
questions and answers from an email:

> Question: By 'training data set of features' do you mean a sample. i.e. including the training and test data just not split up yet?

The idea is to include all the data you have available; the function is going to be splitting it up internally into smaller training and test sets. So, for example, this might be iris[, 1:4].

> Question: you want the labels passed seperately?

Yes, that's what I was proposing. It's a pretty good way to design this. You could imagine passing in a data frame with features and the labels, but then you'd have to have some convention for knowing which column is labels, or a way of specifying which column is labels.

> Question: is the accuracy reached in your mind a vector of accuracies per run or one single number representing the average of all runs?

As written, I was describing a function that returns one number, the average. You could write it to return a vector of accuracies instead and that would be fine; you can always just take the mean() of the result. Either design is pretty good.
